docs/release-notes/2.20.0.md

```diff
@@ -0,0 +1,60 @@
+# Percona Monitoring and Management 2.20.0
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">July 27, 2021</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## Release Highlights
+
+- We've improved the [Easy-install script] for PMM Server by adding checksum verification. Installing PMM on a Linux machine is now even easier.
+- PMM will use Grafana 7.5 instead of 7.3. (Read more at [grafana.com].) We are also preparing for upgrades to Grafana 8.
+- PostgreSQL monitoring with the `pg_stat_monitor` plugin enabled exposes some new metrics, such as Plan Times, WAL Fpi/Bytes/Records.
+- For users who use PMM through the AWS Marketplace, AWS RDS service discovery will be executed without AWS credentials and tuning [IAM roles].
+
+[Easy-install script]: https://www.percona.com/doc/percona-monitoring-and-management/2.x/setting-up/server/easy-install.html
+[grafana.com]: https://grafana.com/docs/grafana/latest/whatsnew/whats-new-in-v7-5
+[IAM roles]: https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html
+
+## New Features
+
+- [PMM-8157](https://jira.percona.com/browse/PMM-8157): Backup Management: Show scheduled backups
```

Comment:
What about something like this instead?

[PMM-8155](https://jira.percona.com/browse/PMM-8155): Backup Management: Support for Backup Scheduling has been added so that users can define backup jobs to run automatically in future times with the option of making the schedules repetitive.
[PMM-8157](https://jira.percona.com/browse/PMM-8157): Backup Management: A new view that shows the list of scheduled backups with quick actions to enable/disable, edit, copy, and delete.

... or something along those lines. I feel that with the current text we aren't highlighting this nice functionality which is part of transitioning Backup Management from TEch Preview into GA.

Reply:
Added ^  to "Additional Release Notes" for the issues, so this will appear in the generated text